### PR TITLE
Update doc about the usage of register_vector/map.

### PR DESCRIPTION
**Pull request by Becavalier**
@chadaustin @sbc100  @kripken Anyone who could help review the grammar? : ) Or add some other instructions?

### CODE REVIEW

File: site/source/docs/porting/connecting_cpp_and_javascript/embind.rst [hunk 1]
```diff
@@ -816,6 +816,83 @@ For convenience, *embind* provides factory functions to register
...
+    EMSCRIPTEN_BINDINGS(module) {
+        class_<xClass>("xClass")
+            .constructor<int, string>()
+            .function("returnVectorData", &xClass::returnVectorData)
+            .function("returnMapData", &xClass::returnMapData);
+        // register the using of container types
+        register_vector<int>("vector<int>");
+        register_map<int, string>("map<int, string>");
+
+We can use the following JavaScript code to interact with the C++ code shows above.
```
Comment (sbc100) [inline site/source/docs/porting/connecting_cpp_and_javascript/embind.rst]: `The following JavaScript can be used to interact with the above C++.`

File: site/source/docs/porting/connecting_cpp_and_javascript/embind.rst [hunk 2]
```diff
@@ -816,6 +816,83 @@ For convenience, *embind* provides factory functions to register
         register_map<int,int>("MapIntInt");
     }
 
+A full example shows below:
```
Comment (sbc100) [inline site/source/docs/porting/connecting_cpp_and_javascript/embind.rst]: `A full example is shown below:`

**Comment by Becavalier** [comment]
Thanks @sbc100!! Updated.

**Comment by Becavalier** [comment]
@chadaustin  Yes, you are right! Doc updated.

**Comment by Becavalier** [comment]
@chadaustin Maybe we could merge this PR? : )

**Comment by kripken** [comment]
Looks like the questions have been addressed here, merging, but if you have time @chadaustin please confirm later.

**Comment by chadaustin** [comment]
Looks great to me! Thank you @Becavalier.

**Pull request merged**